src/CleanActivitylogCommand.php

```diff
@@ -25,13 +26,15 @@ public function handle()
     {
         $this->comment('Cleaning activity log...');
 
+        $log = $this->argument('log');
+
         $maxAgeInDays = config('activitylog.delete_records_older_than_days');
 
         $cutOffDate = Carbon::now()->subDays($maxAgeInDays)->format('Y-m-d H:i:s');
 
         $activity = ActivitylogServiceProvider::getActivityModelInstance();
 
-        $amountDeleted = $activity::where('created_at', '<', $cutOffDate)->delete();
+        $amountDeleted = $activity::where('created_at', '<', $cutOffDate)->inLog($log)->delete();
```

Comment:
this is breaking change, isn't it?

I think you should use something like that
```
$amountDeleted = $activity::where('created_at', '<', $cutOffDate)
    ->when($log !== null, function($query) use ($log) {
            $query->inLog($log);
        })
    ->delete();
```

Because now, if you're passing null to inLog scope, it will looking for activities with NULL log_name

Reply:
Sounds like a good suggestion. Could you make it so @ricardoaugusto ?